### DISCUSSION TITLE
龙猫有没有优化自述文件的计划？

### DISCUSSION THREAD

**Discussion by liubanlaobanzhang** [Meloong-Git/PCL#191] [提问] [answered]
> 通常而言，一个优秀的自述文件会吸引更多的人来关注这个项目……

讲真的……之前看过不少项目，翻阅过不少大佬做的自述文件，都是耳目一新的感觉。

相比于这个仓库“平平无奇”以至于连一个badge都没有的README，我觉得真的差的不是一点半点……（当然我也知道龙猫这个仓库之前被DMCA Takedown掉过，现在能恢复起来已经不错了）

我个人之前倒腾过这些东西……不能说很有经验，但起码能好一点（可以看 https://github.com/liubanlaobanzhang/65-Save 项目里面的README）。。。

> 我也不会像之前某个人在里面乱加Badge（随便加一个WE ARE RICH属实是不大尊敬人）

额 总之 如果龙猫有重写自述文件的计划的话 我就不掺和了 如果没有的话 我过段时间会提交一个PR 这是对自述文件的修改……

（反正我也是用爱发电……）

**Comment by Pigeon0v0** [comment]
怎么又有人内涵我了（

我这次加个正经的吧

**Reply by liubanlaobanzhang** [reply]
……不是啊 我是实在看不下去了 这个 Readme 实在是过于简陋

虽然说PCL的知名度还是挺高的 但是可能还会有 Visual Basic 界的大佬刷到的时候觉得这是个垃圾项目……

而且我不大理解 为什么当时龙猫提出要搞开源的时候的开发组通讯做的那～么好 在这个自述文件上却摆烂了 直接 Copy 过来都行啊……

**Reply by liubanlaobanzhang** [reply]
而且我说的优化**不仅仅局限于加Badge**（当然这个也是优化的一部分） 
它包括排版 引导 甚至说是程序的下载（有些人不知道从哪下载就来 Github 找）等等 整个的版面都要重新设计（太简陋了 没办法……）

**Comment by z0z0r4** [comment, marked as answer]
Readme有很多风格吧…简洁也好
你觉得真有人从GitHub第一次来PCL2？

**Comment by z0z0r4** [comment]
> > 通常而言，一个优秀的自述文件会吸引更多的人来关注这个项目……
> 
> 讲真的……之前看过不少项目，翻阅过不少大佬做的自述文件，都是耳目一新的感觉。
> 
> 相比于这个仓库“平平无奇”以至于连一个badge都没有的README，我觉得真的差的不是一点半点……（当然我也知道龙猫这个仓库之前被DMCA Takedown掉过，现在能恢复起来已经不错了）
> 
> 我个人之前倒腾过这些东西……不能说很有经验，但起码能好一点（可以看 https://github.com/liubanlaobanzhang/65-Save 项目里面的README）。。。
> 
> > 我也不会像之前某个人在里面乱加Badge（随便加一个WE ARE RICH属实是不大尊敬人）
> 
> 额 总之 如果龙猫有重写自述文件的计划的话 我就不掺和了 如果没有的话 我过段时间会提交一个PR 这是对自述文件的修改……
> 
> （反正我也是用爱发电……）

不能说是很会，只能说是不太理解你这《待拍摄》，把本就有点臭长的readme又加了一坨，太抽象了
![Screenshot_20221009_151122_com.github.android.jpg](https://user-images.githubusercontent.com/78744121/194743181-e062da94-977e-4ae9-b3fe-a36fd107250d.jpg)

版权在2. 实在是不知道咋支持
通常CC都在readme末尾/开头吧…![Screenshot_20221009_151156_com.github.android.jpg](https://user-images.githubusercontent.com/78744121/194743204-6311552c-c527-4a2c-b367-67405ea3e85e.jpg)

**Reply by z0z0r4** [reply]
readme还是建议简述吧…
当detail实在是太超模了

**Reply by liubanlaobanzhang** [reply]
那个项目基本算是废了的

PS：麻烦不要再在 PCL2 项目里抨击这个项目的种种问题 我不想因为自己的问题打扰龙猫和其他的社区伙伴🙏
